src/app/space/app-launcher/app-launcher.module.ts

```diff
@@ -2,18 +2,45 @@ import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { Http } from '@angular/http';
 
+import {
+  AuthHelperService,
+  Config,
+  HelperService,
+  TokenProvider
+} from 'ngx-forge';
+
```

Comment:
We have 3 sections separated by blank lines.
1) Angular imports
2) external imports
3) internal imports

Within each section they are alphabetical by the library. Within the braces the components are alphabetical but they have not relationship with the libraries.